### PR TITLE
Enlarged text and changed colors on navbar

### PR DESCRIPTION
**Pull request by S-reinken**
I made the colors for the navbar under our control, and made the text bigger, for usability.

### CODE REVIEW

File: ghu_web/ghu_global/static/ghu_global/css/style.css [hunk 1]
```diff
@@ -44,18 +44,24 @@ nav li {
...
 nav a:hover {
     color: white;
-    background-color: #aaa;
     text-decoration: none;
-    border-bottom-color: #888;
+    border-bottom-color: #002db3;
+}
+
+.nav > li > a:hover, .nav > li > a:focus {
+  background-color: #b3c6ff;
```
Comment (ausbin) [inline ghu_web/ghu_global/static/ghu_global/css/style.css]: Could we stay consistent with four spaces per tab?

File: ghu_web/ghu_global/static/ghu_global/css/style.css [hunk 2]
```diff
@@ -44,18 +44,24 @@ nav li {
...
     text-align: center;
+    font-size: 20px;
     padding: 8px;
-    border-bottom: 4px solid #aaa;
+    border-bottom: 4px solid #668cff;
 }
 
 nav a:hover {
     color: white;
-    background-color: #aaa;
```
Comment (ausbin) [inline ghu_web/ghu_global/static/ghu_global/css/style.css:56]: Combined with the `color: white` above, removing this makes navbar buttons nearly unreadable on hover:

screenshot from 2017-05-21 12-08-18

**Review by ausbin — COMMENTED** [review]
At first I was skeptical of the new navbar, but I'm feeling it now. Only problem I see is that on hover, you can't read the buttons

File: ghu_web/ghu_global/static/ghu_global/css/style.css (hunk 2 above)
Comment (S-reinken) [reply ghu_web/ghu_global/static/ghu_global/css/style.css:56]: Whoa, I thought I changed that on-hover background color to be darker. Let me check.

File: ghu_web/ghu_global/static/ghu_global/css/style.css (hunk 1 above)
Comment (S-reinken) [reply ghu_web/ghu_global/static/ghu_global/css/style.css]: Yeah, you're right, forgot to change it.

File: ghu_web/ghu_global/static/ghu_global/css/style.css (hunk 2 above)
Comment (S-reinken) [reply ghu_web/ghu_global/static/ghu_global/css/style.css:56]: Oh, yeah that's caused by Bootstrap.  It gets fixed when the nav's class is set to "navbar".

Comment (S-reinken) [reply ghu_web/ghu_global/static/ghu_global/css/style.css:56]: scrshot

Comment (S-reinken) [reply ghu_web/ghu_global/static/ghu_global/css/style.css:56]: That should be better.

**Review by ausbin — APPROVED** [review]
lgtm

**Comment by ausbin** [comment]
Sorry for the delay in approving these, I won't do this again

**Comment by S-reinken** [comment]
Okay, I am using this PR to fix what went wrong with the navbar.

**Comment by ausbin** [comment]
lit

**Comment by S-reinken** [comment]
Alright, I am merging this now.  After this, all changes get merged to develop, not master.  Master gets moved when you finish the auto-deployment doop.

**Pull request merged**